scripts/genflagdocs.sh

```diff
@@ -22,6 +22,22 @@ for x in "${bucketCommands[@]}"; do
 done
 
 # Change dir so embedmd understand the local references made in our markdown doc.
-pushd "docs/components"
+pushd "docs/components" > /dev/null
+if [ -z "${1:-}" ]; then
```

Comment:
Can we comment what we expect here exactly and how to control this? It is not obvious from looking just on bash (:

Reply:
Looks like "is there any arg".. Maybe check for `check` in $1 directly?